llvm-external-projects/iree-dialects/lib/Dialect/LinalgExt/Transforms/Transforms.cpp

```diff
@@ -191,6 +191,26 @@ LinalgTileAndFuseTensorOpsBasePattern::returningMatchAndRewrite(
   return tileLoopNest;
 }
 
+/// Peel loops after tiling.
+static void peelTiledLinalgOp(RewriterBase &rewriter,
```

Comment:
Where is this used?

Reply:
A bit below: https://sourcegraph.com/github.com/iree-org/iree/-/blob/llvm-external-projects/iree-dialects/lib/Dialect/LinalgExt/Transforms/Transforms.cpp?L223

Reply:
That method is unused upstream so I dropped it there and this is the only use left.